Plain Craft Launcher 2/Resources/Language/en_US.xaml

```diff
@@ -1,10 +1,1128 @@
+    
+    <!--PageDownloadClient-->
+    <s:String x:Key="LangDownloadCilentLoadingList">Getting version list</s:String>
+
+    <!--PageDownloadCompDetail-->
+    <s:String x:Key="LangDownloadCompToCurseForge">Go to CurseForge</s:String>
+    <s:String x:Key="LangDownloadCompToModrinth">Go to Modrinth</s:String>
+    <s:String x:Key="LangDownloadCompToMCMod">Go to MCMod</s:String>
+    <s:String x:Key="LangDownloadCompCopyName">Replication name</s:String>
+    <s:String x:Key="LangDownloadCompLoadingList">Getting version list</s:String>
+    <s:String x:Key="LangDownloadCompApiSlow">Since version information from the API is updated slowly, it may not be able to recognize the recently updated Minecraft version. Just wait for a while and it will automatically return to normal.</s:String>
+    <s:String x:Key="LangDownloadCompSearchingSuitableInstance">Looking for the right game version…</s:String>
+    <s:String x:Key="LangDownloadCompNoSuitableInstance">No instances suitable for this mod was found in the current Minecraft folder!</s:String>
+
+    <!--PageDownloadFabric-->
+    <s:String x:Key="LangDownloadFabricDescTitle">Introduction to Fabric</s:String>
+    <s:String x:Key="LangDownloadFabricDesc">Fabric Loader is a lightweight mod loader under the new version of Minecraft. You need to install it first before you can install various Fabric mods.&#xa;This page provides Fabric installer downloads. After downloading, you need to manually open the installer to install.</s:String>
+    <s:String x:Key="LangDownloadFabricWebsite">Open official website</s:String>
+    <s:String x:Key="LangDownloadFabricVersionList">Version List</s:String>
+    <s:String x:Key="LangDownloadFabricGettingList">Getting Fabric list</s:String>
+
+    <!--PageDownloadForge-->
+    <s:String x:Key="LangDownloadForgeDescTitle">Introduction to Forge</s:String>
+    <s:String x:Key="LangDownloadForgeDesc">Forge is a mod loader. You need to install Forge first to install various Forge mods.</s:String>
+    <s:String x:Key="LangDownloadForgeWebsite">Open official website</s:String>
+    <s:String x:Key="LangDownloadForgeGettingList">Getting Forge list</s:String>
+    
+    <!--PageDownloadInstall-->
+    <s:String x:Key="LangDownloadInstallReview">Preview of installation content</s:String>
+    <s:String x:Key="LangDownloadInstallFabricApi">You have not chosen to install the Fabric API, which will make most mods unusable!</s:String>
+    <s:String x:Key="LangDownloadInstallOptiFabric">You have not chosen to install OptiFabric, which will make OptiFine unusable!</s:String>
+    <s:String x:Key="LangDownloadInstallOptifine">OptiFine is not compatible with some mods, please install carefully.</s:String>
+    <s:String x:Key="LangDownloadInstallOldOptiFabric">After installation, please search for "OptiFabric Origins" in Mods Download and download it, otherwise "OptiFine" will not be available!</s:String>
```

Comment:
```suggestion
    <s:String x:Key="LangDownloadInstallOldOptiFabric">After the installation, please search for "OptiFabric Origins" in Mods Download and download it, otherwise "OptiFine" will not be available!</s:String>
```